adding_species_manuscript.tex

```diff
@@ -152,10 +152,10 @@
 \section*{Abstract}
 
 Simulation is a key tool in population genetics for both methods development and empirical research,
-but producing simulations that recapitulate even the main features of genomic datasets remains a major obstacle.
+but producing simulations that recapitulate the main features of genomic datasets remains a major obstacle.
 Today, more realistic simulations are possible thanks to large increases in
-the quantity of available data
-and the sophistication of inference and simulation software.
+genetic data,
```

Comment:
I like the increased specificity here, perhaps add "the quantity and quality of available data"

Reply:
I suggest:
> the quantity and quality of available genetic data